### ISSUE TITLE
Poetry does not install packages extras

### ISSUE THREAD

**Issue by mazlum** [python-poetry/poetry#3215]
Hello,

I have started using Poetry.  I installed django-import-export. But poetry does not install tablib exstras. You can see [setup.py](https://github.com/django-import-export/django-import-export/blob/master/setup.py). Django-import-export has tablib[html,ods,xls,xlsx,yaml] requirement. But poetry does not install these(html,ods..) packages.

**Comment by abn** [comment]
@mazlum  I am unable to reproduce this.

```sh
podman run --rm -i --entrypoint bash python:3.8 <<EOF
set -e
pip install -q poetry==1.1.3
poetry new foobar
pushd foobar
sed -i /pytest/d pyproject.toml
poetry add Django-import-export
poetry show -t
EOF
```

```console
Created package foobar in foobar
/foobar /
Creating virtualenv foobar-lWDpn5M1-py3.8 in /root/.cache/pypoetry/virtualenvs
Using version ^2.4.0 for django-import-export

Updating dependencies
Resolving dependencies...

Writing lock file

Package operations: 16 installs, 0 updates, 0 removals

  • Installing defusedxml (0.6.0)
  • Installing et-xmlfile (1.0.1)
  • Installing jdcal (1.4.1)
  • Installing asgiref (3.2.10)
  • Installing markuppy (1.14)
  • Installing odfpy (1.4.1)
  • Installing openpyxl (3.0.5)
  • Installing pytz (2020.1)
  • Installing pyyaml (5.3.1)
  • Installing sqlparse (0.4.1)
  • Installing xlrd (1.2.0)
  • Installing xlwt (1.3.0)
  • Installing diff-match-patch (20200713)
  • Installing django (3.1.2)
  • Installing tablib (2.0.0)
  • Installing django-import-export (2.4.0)
django-import-export 2.4.0 Django application and library for importing and exporting data with included admin integration.
|-- diff-match-patch *
|-- django >=2.0
|   |-- asgiref >=3.2.10,<3.3.0 
|   |-- pytz * 
|   `-- sqlparse >=0.2.2 
`-- tablib >=0.14.0
    |-- markuppy * 
    |-- odfpy * 
    |   `-- defusedxml * 
    |-- openpyxl >=2.6.0 
    |   |-- et-xmlfile * 
    |   `-- jdcal * 
    |-- pyyaml * 
    |-- xlrd * 
    `-- xlwt * 
```

**Comment by mazlum** [comment]
Ohh. I'm sorry. I was using poetry 1.1.2. I think it was fixed 1.1.3. I updated poetry and problem solved. 
Thank you very much.
